### ISSUE TITLE
SyntaxError: invalid syntax

### ISSUE THREAD

**Issue by Arxdust** [KristobalJunta/rss-to-telegram#1]
When start main.py i have error

python3 main.py
Traceback (most recent call last):
  File "main.py", line 5, in <module>
    from config import config
  File "/home/......./rss3/config.py", line 8
    "channel-id": '119******32'
                ^
SyntaxError: invalid syntax

Advise how to fix it

**Comment by KristobalJunta** [comment]
@Arxdust can you please supply a full snippet of your config?

**Comment by Arxdust** [comment]
@KristobalJunta Sorry I found problem, this is my error, i deleted "," after TOKEN

But now i have another error
main.py - original
config.py - modifited (change only token, channel-id, rss)

python3 main.py
Traceback (most recent call last):
  File "main.py", line 48, in <module>
    if entry['guid'] not in guids:
  File "/usr/local/lib/python3.5/dist-packages/feedparser.py", line 357, in __getitem__
    return dict.__getitem__(self, key)
KeyError: 'guid'

**Comment by KristobalJunta** [comment]
Yes, I was supposing it as a problem with commas or smth like that.

Regarding the guid issue. That was the field in the instance of RSS feed I used to parse. I used guid as a unique field to keep track of already posted items. Feel free to modify code to use any other field or other way of keeping track of it.
Also feel free to pull request if you come up with a more smart way to address this.
